test/grouped_convnd_bwd_weight/test_grouped_convnd_bwd_weight.cpp

```diff
@@ -27,28 +29,59 @@ class TestGroupedConvndBwdWeight : public ::testing::Test
     using NDimSpatial = std::tuple_element_t<6, Tuple>;
 
     std::vector<ck::utils::conv::ConvParam> conv_params;
-    ck::index_t split_k{2};
+    std::vector<ck::index_t> split_ks{1, 2};
+
+    bool skip_case(const ck::utils::conv::ConvParam& params, const ck::index_t split_k)
+    {
+        // K=1 or C=1 is supported only by dl kernel
+        // dl kernel is only supported for split_k=1
```

Comment:
just to make it consistent, here and below
```suggestion
        // K=1 or C=1 is supported only by DL kernel
        // DL kernel is only supported for split_k=1
```

Reply:
Fixed.